src/Raven.Server/Documents/ETL/EtlTransformer.cs

```diff
@@ -86,12 +86,12 @@ private JsValue LoadToFunctionTranslator(JsValue self, JsValue[] args)
             if (args[1].IsObject() == false)
                 ThrowInvalidScriptMethodCall("loadTo(name, obj) second argument must be an object");
 
-            using (var result = new ScriptRunnerResult(DocumentScript, args[1].AsObject()))
-            {
-                LoadToFunction(args[0].AsString(), result);
-
-                return result.Instance;
-            }
+            // explicitly not disposing here, this will clear the context from the JavaScriptUtils, but this is 
```

Comment:
This is also releasing all disposables (cloned documents). Who is releasing them now?